test-it/ViaTests/InRangeAndCoerceNaN.via

```diff
@@ -47,25 +43,13 @@ define (%3A%3AWeb%20Server%3A%3AInteractive%3A%3AWebApp%3A%3AInRangeAndCoerceNaN
         clump(1
         InRangeAndCoerce(c2 c1 c0 local34 local33 local10 local9)
         Println(local10)
-        IsNotANumPathRefnum(local10 local11 )
```

Comment:
Checking for a NaN with IsNotANumPathRefum primitive is still valid and need not be removed, I think. dataItem_Pass at the end could be printed and checked for.

Reply:
My reasoning was that printing out the values of the variables (that they equal "NaN") was equivalent to what the IsNotANumPathRefnum was doing, and checking the dataItem_Pass at the end provides no additional value over the test as a whole, since if dataItem_Pass was false it would imply that one of the variables was not what I was expecting, meaning that the results printed were different than in the .vtr file. Does that make sense?